### PR TITLE
Extend permissions in run namespace

### PR DESCRIPTION
**Pull request by rinckm**
### Description

The permissions of the default service account in run namespaces are extended by full CRUD permissions on:
- configmaps
- secrets

### Submitter checklist

- [x] Change has been tested (on a back-end cluster)
- [x] (If applicable) Jira backlog item ID added to the PR title and the [changelog.yaml] entry
- [x] [changelog.yaml] entry with upgrade notes is prepared and appropriate for the audience affected by the change (users or developer, depending on the change)
- [x] Semantic version diffed against [last release][releases] and updated accordingly. In this project the version has to be maintained here:
    - [/charts/steward/Chart.yaml](https://github.com/SAP/stewardci-core/blob/master/charts/steward/Chart.yaml) (`version` and `appVersion`)

In case dependencies have been updated:
- [ ] Links to external changelogs, since the last release of our component, added to the [changelog.yaml] entry (description).
- [ ] Changelogs read thoroughly, potential impact described, upgrade notes prepared (if necessary)
- [ ] Check if dependency updates affect our semantic version increment.

### Reviewer checklist

Before the changes are marked as `ready-for-merge`: 

- [x] There is at least 1 approval for the pull request and no outstanding requests for change
- [x] All voter checks have passed
- [x] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [x] The Pull Request title is understandable and reflects the changes well
- [x] The Pull Request description is understandable and well documented
- [x] [changelog.yaml] entry for this Pull Request has been added
    - [x] Changelog entry contains all required information
    - [x] 'Upgrade notes' are documented in changelog.yaml (if required)

[changelog.yaml]: https://github.com/SAP/stewardci-core/changelog.yaml
[releases]: https://github.com/SAP/stewardci-core/releases

### CODE REVIEW

File: changelog.yaml [hunk 1]
```diff
@@ -52,6 +52,17 @@
...
 
+    - type: enhancement
+      impact: minor
+      title: Extend permissions of default user in a run namespace
+      description: |-
+        The permissions of the default user in its run namespace are extended.
+        The default user has now full CRUD permissions on:
+        - configmaps
+        - secrets
+      pullRequestNumber: 0
```
Comment (rinckm) [inline changelog.yaml:63]:
```suggestion
      pullRequestNumber: 324
```

**Review by romanisb — APPROVED** [review]

File: changelog.yaml [hunk 2]
```diff
@@ -52,6 +52,17 @@
   date: TBD
   changes:
 
+    - type: enhancement
+      impact: minor
+      title: Extend permissions of default user in a run namespace
```
Comment (anbrsap) [inline changelog.yaml:57]:
```suggestion
      title: Extend permissions of default service account in run namespaces
```

File: changelog.yaml [hunk 3]
```diff
@@ -52,6 +52,17 @@
...
+      description: |-
+        The permissions of the default user in its run namespace are extended.
```
Comment (anbrsap) [inline changelog.yaml:59]: Ditto

File: charts/steward/templates/clusterrole-run.yaml [hunk 4]
```diff
@@ -9,13 +9,13 @@ rules:
   resources: ["pods"]
   verbs: ["create","delete","get","list","patch","update","watch"]
 - apiGroups: [""]
-  resources: ["pods/exec"]
+  resources: ["configmaps","pods/exec","secrets"]
```
Comment (anbrsap) [inline charts/steward/templates/clusterrole-run.yaml:12]: Better use a dedicated entry per resource.

File: changelog.yaml [hunk 5]
```diff
@@ -52,6 +52,17 @@
...
   changes:
 
+    - type: enhancement
+      impact: minor
+      title: Extend permissions of default service account in run namespaces
+      description: |-
+        The permissions of the default service account in run namespaces are extended.
+        The default user has now full CRUD permissions on:
+        - configmaps
+        - secrets
```
Comment (anbrsap) [inline changelog.yaml:62]:
```suggestion
        The permissions of the default service account in run namespaces are extended
        by full CRUD permissions on:
        - configmaps
        - secrets
```

File: changelog.yaml [hunk 6]
```diff
@@ -52,6 +52,17 @@
...
+      pullRequestNumber: 324
+      jiraIssueNumber: CLOUDCI-1609
```
Comment (anbrsap) [inline changelog.yaml:64]:
```suggestion
      jiraIssueNumber: 1609
```

**Review by anbrsap — APPROVED** [review]

**Pull request merged**